lib/committee/schema_validator/option.rb

```diff
@@ -46,6 +47,7 @@ def initialize(options, schema, schema_type)
         @optimistic_json                  = options.fetch(:optimistic_json, false)
         @parse_response_by_content_type   = options.fetch(:parse_response_by_content_type, true)
         @parameter_overwite_by_rails_rule = options.fetch(:parameter_overwite_by_rails_rule, true)
+        @permit_blank_structures          = options.fetch(:permit_blank_structures, false)
```

Comment:
nits) As for the name of the parameter, how about `allow_blank_structures` to match the name of the other parameter if you want to allow it?

https://github.com/interagent/committee/blob/d1488a8d11cd323753e42587c48b45285640494f/lib/committee/schema_validator/option.rb#L42-L43

Reply:
Thank you.
I have renamed permit_blank_structures to allow_blank_structures.

ref: https://github.com/interagent/committee/pull/417/commits/6de758350e2a7d0b1a7a49720637451d177cf8a3